packages/ketcher-core/src/domain/entities/text.ts

```diff
@@ -28,8 +28,16 @@ export enum TextCommand {
 export interface TextAttributes {
   // TODO: add Interface for content type
   content: string
-  position: Vec2
-  pos: Array<Vec2>
+  position: Point
+  pos: Array<Point>
+}
+
+function preparePositions(positions) {
+  if (!positions || positions === []) {
```

Comment:
if (!positions || !positions.length) { ...

<img width="931" alt="Screenshot 2022-10-13 at 13 59 11" src="https://user-images.githubusercontent.com/112183450/195552325-aa50488a-6a7e-4839-a314-696a269d263a.png">

Reply:
Fix, thank you!